libmc/_client.pyx

```diff
@@ -1024,6 +1050,9 @@ cdef class PyClient:
         self._record_thread_ident()
         return self.touch(key, -1)
 
+    def update_servers(self, servers):
+        return self._update_servers(servers)
```

Comment:
if use cpdef update_servers instead，no need to both "def update_servers" "cdef _update_servers" ref: http://notes-on-cython.readthedocs.io/en/latest/function_declarations.html#cpdef-it-s-both

Reply:
or just "def update_servers", no need to call it from cython inner?